src/Wpf.Ui/Controls/Button/Button.cs

```diff
@@ -13,16 +13,16 @@ namespace Wpf.Ui.Controls;
 /// </summary>
 /// <example>
 /// <code lang="xml">
-/// &lt;ui:Button
```

Comment:
Examples should not use special characters
https://wpfui.lepo.co/api/Wpf.Ui.Controls.Button.html

Reply:
Because? I mean even vs is using it totally fine.

Reply:
Oh and btw & is a special character :)